README.md

```diff
@@ -1 +1,112 @@
-# react-capstone-metrics-webapp
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Metrics Webapp - Jose Abel Ramirez Frontany
+
+React Capstone where I have to build about building a mobile web application to check a list of metrics (numeric values) making use of React and Redux. I selected an API that provides numeric data about a topic that I like and then build the webapp around it. I choose the API of "CoinGecko API V3".
+
+The webapp have 2 pages:
+
+- The first page with a list of items that could be filtered by some parameters; in my implementation, it's a list of cryptocurrencies that can be filtered by the name of the cryptocurrency. This page is my homepage.
+
+- The second page for the item details; in my implementation, the detail page routes to the details of a particular cryptocurrency once is clicked.
+
+![screenshot](./app_screenshot.png)
+![screenshot](./app_screenshot2.png)
+
+## Built With
+
+- React
+- React-dom
+- Redux
+- Redux-Thunk
+- React-Router
+- Fetch
+- Jest
+- HTML
+- CSS
+
+## Documentation
+
+- https://www.coingecko.com/en/api/documentation
+- https://testing-library.com/docs/dom-testing-library/
+
+## Video
+
+https://www.loom.com/share/a526a867f90242fb963601b318b877a5
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
+### Setup
+
+Go to the top of the page, press te green button that says "Code", and copy the link. Then you have to go to your console and type
+
+```
+    git clone 'repository-link'
+```
+
+That's all, you are ready to go!
+
+# Live Demo
+
+[Metrics Web App](https://jose-abel-metrics-webapp.netlify.app/)
```

Comment:
- [ ] Good job implementing the changes requested. Locally, they work fine now. However, looking at the image below, your changes are not pushed to your live demo yet. Kindly push them.

![jose-livedemo](https://user-images.githubusercontent.com/36542986/142300245-b62f0b84-ab95-421d-a4f4-a564f149253a.jpg)

Reply:
These issues are well resolved! ✔️